services/port_manager/src/test/java/com/futurewei/alcor/portmanager/controller/PortControllerTest.java

```diff
@@ -0,0 +1,335 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.portmanager.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.futurewei.alcor.portmanager.config.UnitTestConfig;
+import com.futurewei.alcor.portmanager.repo.PortRepository;
+import com.futurewei.alcor.web.entity.port.*;
+import com.futurewei.alcor.web.entity.mac.*;
+import com.futurewei.alcor.web.entity.vpc.*;
```

Comment:
* vpc/port/mac/ip/mac are in their own sub-dictionary in com.futurewei.alcor.web.entity. 
* subnet and route are still one level up. 

Let us create the same sub-dictionary for subnet and route. Later we could ask other microservices to use the new web classes path and retire the existing ones.

Reply:
subnet and route are still one level up.--->This will cause other microservices to fail to compile. or I need to modify the code of other microservices. Can I do this in the next PR?

Reply:
I would suggest you to add two new sub-dictionaries, one for subnet and the other for route, copy those web classes to the new dictionaries, and reference only in PortManager.

We would ask other microservice owners to make the change pointing to the new classes and then remove old/existing ones.

Reply:
Got it.